setup.py

```diff
@@ -15,4 +15,7 @@
     version='8.20.1.7',
     python_requires='>=3.5',
     packages=packages,
+    package_data={'archinfo': ['py.typed']},
+    test_suite='nose.collector',
+    tests_require=['nose'],
```

Comment:
I'm not sure that we want these two lines, at least as part of the PR.

Reply:
That was probably some rebase related messup and definitely not intentional, will fix.